boot/boot.go

```diff
@@ -400,18 +400,18 @@ func SetRecoveryBootSystemAndMode(dev Device, systemLabel, mode string) error {
 // UpdateManagedBootConfigs updates managed boot config assets if those are
 // present for the ubuntu-boot bootloader. Returns true when an update was
 // carried out.
-func UpdateManagedBootConfigs(dev Device) (updated bool, err error) {
+func UpdateManagedBootConfigs(dev Device, gadgetSnapOrDir string) (updated bool, err error) {
```

Comment:
(nitpick^2) maybe `gadgetSnapFileOrDir` or even `gadgetDirOrSnapPath` (which is what we use in the boot/cmdline.go where this eventually will be passed to. But I don't think it's worth updating this PR for this, it could be done in one of the followups.